.vib/solr/cypress/cypress/integration/utils.js

```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+
+export let random = (Math.random() + 1).toString(36).substring(7);
+console.log("random", random);
```

Comment:
This can be removed. I think this has been copied/pasted from another test. I will fix the origin as well.

Reply:
Done